core/components/molecules/fileList/__stories__/index.story.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import { FileList, Button } from '@/index';
+import { fileList } from './__common__/fileListExample';
+import { action } from '@storybook/addon-actions';
+
+export const all = () => {
+
+  const onClick = (file: File, id: any) => {
+    return action(`Clicked ${file} at index ${id}`)();
+  };
+
+  return (
+    <FileList
+      onClick={onClick}
+      fileList={fileList}
+      className="mt-4 FileList"
+      actionRenderer={fileItem => {
+        if (fileItem.id === 3) {
```

Comment:
Why is id hardcoded?

Reply:
Just an example case showing how we can render different actions on different file items using a unique property of fileItem ( in this case id ).